docs/user-guide/topics/readiness.md

```diff
@@ -0,0 +1,67 @@
+---
+template: main.html
+---
+
+# Checking readiness of Kubernetes resources
+
+When Iter8 experiment rely on Kubernetes resources, the readiness of those resources can be checked first.  
```

Comment:
Many Iter8 experiments, like `load-test-http`(with Kubernetes usage link), and `load-test-grpc` (with Kubernetes usage link) are intended to run inside a Kubernetes cluster and interact with Kubernetes resources. In such cases, it is useful to check the resources involved are *ready* at the beginning of the experiment, before other experiment tasks are executed. You can do this with various `ready.*` experiment parameters.

For example, ...

Reply:
nice. done.